packages/nextjs/package.json

```diff
@@ -79,7 +79,7 @@
     "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/instrumentation-http": "0.53.0",
     "@opentelemetry/semantic-conventions": "^1.27.0",
-    "@rollup/plugin-commonjs": "26.0.1",
```

Comment:
just to clarify, are there no breaking changes here that affect us (e.g. node version support etc)?

Reply:
I didn't find anything, theres no changelog etc. so I relied on our own tests here

Reply:
This stuff is thoroughly tested.